gdc_client/utils.py

```diff
@@ -0,0 +1,14 @@
+from urllib import urlencode
+
+
+def url_with_params(path, *params, **kwparams):
```

Comment:
Can you rename this to something like `build_url` or something more in the line of what the code does, `url_with_params` was a unclear when I first saw it. Also, can you add unit tests for it and some docstring maybe

Reply:
will do